target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -201,6 +201,11 @@ ramips_setup_interfaces()
 	wl-330n)
 		ucidef_set_interface_lan "eth0.1"
 		;;
+	mr200)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "6t@eth0"
+		ucidef_set_interface_wan "usb0"
+		;;
```

Comment:
Unfortunately I am using this modem in production now and so can not easily use it to re-test the settings. it is however possible that i've made a mistake the first time and the default configuration works correctly. If it works in your case, i'm sure it should be fine.